src/components/Layout/LayoutSidebar.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import PropTypes from '@dhis2/prop-types'
+import styles from './LayoutSidebar.module.css'
+
+const LayoutSidebar = ({ children }) => (
+    <aside className={styles.sidebar}>{children}</aside>
```

Comment:
Should be ok right?

> aside is now acceptable for secondary content when not nested within an article element.
> http://html5doctor.com/aside-revisited

> The HTML `<aside>` element represents a portion of a document whose content is only indirectly related to the document's main content. Asides are frequently presented as sidebars or call-out boxes.
> https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside

I find the html5 tag semantics pretty vague tbh, but the usage seems to match the definitions above the way I interpret them.

Reply:
> `whose content is only indirectly related to the document's main content.`

To me the content in die sidebar is directly related to the main content. A sidebar navigation's content does not really relate to the main content, but it does bring you there, so it's indirectly related

Reply:
> To me the content in die sidebar is directly related to the main content.

Ok yeah. Yeah I think that that's where we're getting into a discussion of semantics. I can just change it to a neutral div, since it seems like both our interpretations are a bit different?

Reply:
@ismay @Mohammer5 : note that the sequence in this thread got messed up. I made my comment whilst reviewing, so hadn't seen Ismay's latest comment yet. And Ismay's comment appears later in the thread, but was made earlier.

Reply:
I think we can leave it as an aside then, we not doing SEO, so it does not matter too much

Reply:
> note that the sequence in this thread got messed up. I made my comment whilst reviewing, so hadn't seen Ismay's latest comment yet. And Ismay's comment appears later in the thread, but was made earlier.

Yeah that's weird.